faq.py

```diff
@@ -0,0 +1,114 @@
+'''
+
+FAQ['How do I use BigQuery to write custom queries over the data?'] = \
+'''Check out [Getting Started Accessing the HTTP Archive with BigQuery](https://github.com/HTTPArchive/httparchive/blob/master/docs/bigquery-gettingstarted.md),
+a guide for first-time users written by [Paul Calvano](https://twitter.com/paulcalvano).
+
+For a guided walkthrough of the project, watch this in-depth [30 minute video](https://www.youtube.com/watch?v=00f9kza3BJ0)
+featuring HTTP Archive maintainer [Rick Viscomi](https://twitter.com/rick_viscomi).
+
+If you have any questions about using BigQuery, reach out to the HTTP Archive community
+at [discuss.httparchive.org](https://discuss.httparchive.org/).'''
+
+FAQ[CHANGELOG_QUESTION] = 'Loading&hellip;'
+
+FAQ['What are the limitations of this testing methodology?'] = \
+'''The HTTP Archive examines each URL in the list, but does not crawl the website's other pages. 
+Although this list of websites is well known, the entire website doesn't necessarily map well to a single URL.
+
+Most websites are comprised of many separate web pages. The landing page may not be representative of the overall site.
+Some websites, such as [facebook.com](http://www.facebook.com/), require logging in to see typical content.
+Some websites, such as [googleusercontent.com](http://www.googleusercontent.com/), don't have a landing page. 
+Instead, they are used for hosting other URLs and resources. 
+In this case googleusercontent.com is the domain path used for resources inserted by users into Google documents, etc.
+Because of these issues and more, it's possible that the actual HTML document analyzed is not representative of the website.'''
+
+FAQ['Who sponsors the HTTP Archive?'] = \
+'''The HTTP Archive is sponsored by companies large and small in the web industry who are dedicated to
+moving the web forward. Our sponsors make it possible for this non-profit project to continue operating
+and tracking how the web is built.
+
+See the full list of [HTTP Archive sponsors](/about#sponsors).
+'''
+
+FAQ['How do I make a donation to support the HTTP Archive?'] = \
+'''The HTTP Archive is part of the Internet Archive, a 501(c)(3) non-profit.
+Donations in support of the HTTP Archive can be made through the Internet Archive's [donation page](https://archive.org/donate/).
+Make sure to send a follow-up email to 
+[donations@archive.org](mailto:donations@archive.org?subject=HTTP+Archive+donation) 
+designating your donation to the "HTTP Archive".
+'''
+
+FAQ['Who maintains the HTTP Archive?'] = \
+'''[Steve Souders](https://twitter.com/Souders) created the HTTP Archive in 2010.
+It's built on the shoulders of [Pat Meenan](https://twitter.com/patmeenan)'s 
+[WebPageTest](https://webpagetest.org/) system. 
+We've received patches from several individuals including Jonathan Klein, Yusuke Tsutsumi, Carson McDonald, James Byers, Ido Green, Charlie Clark, Jared Hirsch, and Mike Pfirrmann. 
```

Comment:
To make sure this list is up to date, could we link to the contributions page on GitHub? Makes everyone's life simpler?

Reply:
👍